hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/index.less

```diff
@@ -19,14 +19,11 @@
 body {
   margin: 0;
   padding: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Roboto', 'Segoe UI',
-    'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans',
-    'Helvetica Neue', sans-serif;
+  font-family: 'Roboto', sans-serif;
```

Comment:
We should probably leave this untouched. And override this in the class's CSS instead if needed.